### PR TITLE
i18n(es): Translated `project-structure.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Translated `project-structure.mdx`

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/guides/project-structure.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- `src/content/docs/` — Archivos de contenido. Starlight convierte cada archivo `.md`, `.mdx` o `.mdoc` en este directorio en una página de tu sitio.
+- `src/content/i18n/` (opcional) — Datos de traducción para soportar la [internacionalización](/guides/i18n/).
+- `src/` — Otros códigos fuente y archivos (componentes, estilos, imágenes, etc.) para tu proyecto.
+- `public/` — Recursos estáticos (fuentes, favicon, PDF, etc.) que no serán procesados por Astro.
+
+## Ejemplo de contenido de un proyecto
+
+Un directorio de proyecto de Starlight podría lucir así:
+
+import FileTree from '../../../components/file-tree.astro';
```
Comment (delucis) [inline docs/src/content/docs/es/guides/project-structure.mdx:23]: One more level up here needed to fix the build error I think:

```suggestion
import FileTree from '../../../../components/file-tree.astro';
```

**Review by delucis — COMMENTED** [review]
Translation looks great to me. Will just fix one thing to make the build work.

**Review by delucis — APPROVED** [review]
Great — that worked 😀 

Thanks @Waxer59 & @dreyfus92!

**Pull request merged**